contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -3229,6 +3226,54 @@ class tsqlMutator : public TSqlParserBaseListener
 	{
 		handleModuloOperator(ctx);
 	}
+	
+	void enterUnary_op_expr(TSqlParser::Unary_op_exprContext *ctx) override
+	{
+		/* 
+		 * The T-SQL grammar allows an arbitrary number of unary '+' operators to precede an expression, 
+		 * but PG only supports that for numeric expressions. For string expressions, such a '+' will raise an error in PG.
+		 * In SQL this shows as redundant operators, for example for concatenation: SELECT 'a' ++ 'b'. Expressions
+		 * such as +++(+++@v)) are also valid syntax according to the T-SQL grammar even though they look unusual.
+		 * Here we remove such unary '+' operators, which are redundant anyway. 
+		 * However we do not touch numeric constants (e.g. +123) since the '+', although still redundant, may
+		 * have been included for code clarity (e.g. +123 as opposed to -123).
+		 */
+		std::string op = getFullText(ctx->op);	
+		if (op.front() == '+') {	
+			auto rhsctx = ctx->expression();
```

Comment:
What does rhs stands for in this context ?

Reply:
RHS=right-hand side ; LHS=left-hand side (of an expression/equation)